client/src/components/home-layout/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Layout } from 'antd';
+import './style.scss';
+import MainHeader from '../MainHeader';
+import Sidebar from '../Sidebar';
+import ChatComponent from '../ChatComponent';
+
+const { Header } = Layout;
+
+// eslint-disable-next-line react/prefer-stateless-function
+class HomeLayout extends React.Component {
```

Comment:
why? You are not using state, so you can remove classes. Also, you are using hooks, so that you can remove classes for sure.

Reply:
Converted to stateless component.